nebula/ui/components/MZStepProgressBar.qml

```diff
@@ -70,10 +70,9 @@ Item {
                     }
                 }
 
-                accessibleName: currentState === MZStepProgressBarDelegate.State.Complete ? MZI18n.OnboardingProgressBarAccessibilityStepComplete.arg(labelText).arg(index + 1).arg(progressBar.model.count)
-                                                                                           : MZI18n.OnboardingProgressBarAccessibilityStepCurrent.arg(labelText).arg(index + 1).arg(progressBar.model.count)
-
                 onClicked: activeIndex = index
+
+                accessibleName: currentState === MZStepProgressBarDelegate.State.Complete ? MZI18n.OnboardingProgressBarAccessibilityStepComplete.arg(labelText).arg(index + 1).arg(progressBar.model.count)                                                                                           : MZI18n.OnboardingProgressBarAccessibilityStepCurrent.arg(labelText).arg(index + 1).arg(progressBar.model.count)
```

Comment:
Looks like the formatting of this line needs to be fixed, There sees to be a lot of white space after '?'. Should this be broken across lines?